Dockerfile

```diff
@@ -9,14 +8,12 @@ RUN apt-get update && apt-get -y upgrade && \
         # git is required for installing gems from git repos
         git \
         # libjemalloc1 (v3) provides big memory savings vs jemalloc v5+ (default on debian buster)
-        libjemalloc1 \
         libpq-dev \
-        nodejs \
+        postgresql-client-11 \
```

Comment:
Wondering why we need the explicit client v11 here vs the libpq-dev package.. What is the difference between libpq-dev and the new pg client? 

Also why did we drop node support...isn't that needed for the asset precompilation? will this image build correctly without it?